src/main.py

```diff
@@ -148,9 +149,11 @@ def open_pr(subject, body, branch):
         log.debug("origin repo not writable; creating fork")
         fork = user.create_fork(origin_repo)
         remote = user.login
-        check_call(("git", "remote", "add", remote, fork.ssh_url))
         repo = fork
 
+    remote_url = f"https://{github_token}:x-oauth-basic@github.com/{repo.full_name}"
+    check_call(("git", "remote", "set-url", remote, remote_url))
```

Comment:
I think this will fail on the `remote = user.login` branch since the remote won't've been created.